mw-webapp/src/component/modal/ModalContent/ModalContent.module.scss

```diff
@@ -24,7 +24,7 @@ $modal-animation-time: 150ms;
   overflow: auto;
   max-width: 95vw;
   max-height: 90vh;
-  padding: 20px 45px;
+  padding: $paddingExtraBig;
```

Comment:
ON 28 line I see a border 2 px. Please check do we need it?  Because we don't have such token) Maybe we don't need border at all?

Reply:
done